bin/disco_vpc_ui.py

```diff
@@ -8,7 +8,10 @@
 import argparse
 import sys
 
-from disco_aws_automation import DiscoVPC
+from disco_aws_automation import (
+    DiscoVPC,
+    DiscoVPCPeerings
```

Comment:
question: so the idea is VPC life cycle and VPC  peering life cycle are two different concepts from now on?